_includes/sections/instant-messenger.html

```diff
@@ -1,83 +1,270 @@
+  labels="warning:Inconsistent E2EE:VoIP and file transfers/names may not be end-to-end encrypted."
+  website="https://xmpp.org/"
+  forum="https://forum.privacytools.io/t/discussion-xmpp/2112"
+  windows="https://gajim.org/"
+  mac="https://gajim.org/"
+  linux="https://gajim.org/"
+  fdroid="https://f-droid.org/en/packages/eu.siacs.conversations"
+  googleplay="https://play.google.com/store/apps/details?id=eu.siacs.conversations"
+  ios="https://siskin.im/"
+%}
+
+<div>
+    <h4>Worth Mentioning</h4>
 
-{% include cardv2.html
-title="Signal"
-image="/assets/img/tools/Signal.png"
-description="Signal is a mobile app developed by Open Whisper Systems. The app provides instant messaging, as well as voice and video calling.
-All communications are end-to-end encrypted. Signal is free and open source."
-labels="success:VoIP"
-website="https://signal.org/"
-forum="https://forum.privacytools.io/t/discussion-signal/664"
-github="https://github.com/signalapp"
-windows="https://signal.org/download/"
-mac="https://signal.org/download/"
-linux="https://signal.org/download/"
-googleplay="https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms"
-android="https://signal.org/android/apk/#apk-danger"
-ios="https://apps.apple.com/us/app/signal-private-messenger/id874139669"
+    <ul>
+        <li><a href="https://www.kontalk.org">Kontalk</a> is a community-driven instant messaging network based on XMPP.</li>
+        <li>Other <a href="https://matrix.org/clients">Matrix</a> clients, that may however be less feature complete than Riot.im.</li>
+        <li>Other <a href="https://omemo.top">OMEMO</a> capable clients for XMPP.</li>
+    </ul>
+
+    <h2 id="peer-to-peer" class="anchor"><a href="#peer-to-peer"><i class="fas fa-link anchor-icon"></i></a> Peer to Peer (P2P)</h2>
+
+    <p>Peer-to-Peer instant messengers connect directly to each other without requiring third-party servers. Clients (peers) usually find each other through the use of a <a href="https://en.wikipedia.org/wiki/Distributed_computing">distributed computing</a> network. Examples of this include <a href="https://en.wikipedia.org/wiki/Distributed_hash_table">DHT (distributed hash table)</a> (used with technologies like torrents and IPFS, for example), or or <a href="https://en.wikipedia.org/wiki/Ethereum">Ethereum</a>'s <a href="https://github.com/ethereum/wiki/wiki/Whisper">Whisper</a> protocol (used with some newer DApps). Another approach is proximity based networks, where a connection is established over WiFi or Bluetooth (for example, Briar or the <a href="https://www.scuttlebutt.nz">Scuttlebutt</a> social networking protocol). Once a peer has found a route to its contact via any of these methods, a direct connection between them is made.</p>
```

Comment:
```suggestion
    <p>Peer-to-Peer instant messengers connect directly to each other without requiring third-party servers. Clients (peers) usually find each other through the use of a <a href="https://en.wikipedia.org/wiki/Distributed_computing">distributed computing</a> network. Examples of this include <a href="https://en.wikipedia.org/wiki/Distributed_hash_table">DHT (distributed hash table)</a> (used with technologies like torrents and IPFS, for example), or <a href="https://en.wikipedia.org/wiki/Ethereum">Ethereum</a>'s <a href="https://github.com/ethereum/wiki/wiki/Whisper">Whisper</a> protocol (used with some newer DApps). Another approach is proximity based networks, where a connection is established over WiFi or Bluetooth (for example, Briar or the <a href="https://www.scuttlebutt.nz">Scuttlebutt</a> social networking protocol). Once a peer has found a route to its contact via any of these methods, a direct connection between them is made.</p>
```

Double or looks wrong to me.